Client/Frontend/Settings/SiriSettingsViewController.swift

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import Intents
+import IntentsUI
+
+@available(iOS 12.0, *)
+class SiriSettingsViewController: SettingsTableViewController {
+    let prefs: Prefs
+
+    init(prefs: Prefs) {
+        self.prefs = prefs
+        super.init(style: .grouped)
+
+        self.title = Strings.SettingsSiriSectionName
+        hasSectionSeparatorLine = false
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func generateSettings() -> [SettingSection] {
+        let setting = SiriOpenURLSetting(settings: self)
+        let firstSection = SettingSection(title: nil, footerTitle: NSAttributedString(string: Strings.SettingsSiriSectionDescription), children: [setting])
+        return [firstSection]
+    }
+}
+
+class SiriOpenURLSetting: Setting {
```

Comment:
See above comment about @ available for class. It also looks like you're switching between making some of the Siri classes available with iOS 12 and others having checks inside. Unless there is a reason for this that I'm not picking up on, I'd recommend keeping it consistent.